src/main/java/Command/AddCommand.java

```diff
@@ -0,0 +1,59 @@
+package command;
+
+import exceptions.InvalidCommandException;
+import exceptions.NoDateException;
+import exceptions.NoTaskException;
+import exceptions.WrongDateTimeFormatException;
+import parserstorageui.Storage;
+import parserstorageui.Ui;
+import task.TaskList;
+
+public class AddCommand extends Command {
+
+    /**
+     * Initializes AddCommand class
+     *
+     * @param command
+     */
+    public AddCommand(String command) {
+        super(command);
+    }
+
+    /**
+     * Executes the command
+     **/
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) throws NoDateException, InvalidCommandException,
+        NoTaskException, WrongDateTimeFormatException {
+        try {
+            tasks = tasks.add(this.command);
+        } catch (WrongDateTimeFormatException e) {
+            throw new WrongDateTimeFormatException(e.getMessage());
+        }
+        return ui.showAddedTask(tasks.taskSize(), tasks.getAddedOrDeletedTask());
+    }
+
+    /**
+     * Check if the current command is an exit command
+     **/
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Maybe it is better to name the object with another better name that describes the object better?